src/api/lib/db/queries/index.js

```diff
@@ -47,7 +47,7 @@ const base = (client, tableName, exposedFields, searchableFieldsList, sortableFi
 
     return {
         selectAll: select.selectAll,
-        selectOne: select.selectOneById,
+        selectOne: select.selectOneById, // Not sure that's a great id !!!
```

Comment:
:+1: for the comment pun!

I don't think however we should create some aliases. This is a boilerplate after all. New devs should learn repositories names. :)